CHANGELOG.md

```diff
@@ -11,14 +11,14 @@ The minor version will be incremented upon a breaking change and the patch versi
 ## [Unreleased]
 
 ### Features
-
 - ts: Add optional `commitment` parameter to `Program.addEventListener` ([#3052](https://github.com/coral-xyz/anchor/pull/3052)).
 
 ### Fixes
 
 - idl: Make safety comment checks fail silently when program path env is not set ([#3045](https://github.com/coral-xyz/anchor/pull/3045])).
 
 ### Breaking
+- lang: Add tokio support to `RequestBuilder` to `async` feature in `anchor_client` ([#3057](https://github.com/coral-xyz/anchor/pull/3057])).
```

Comment:
Extra line due to formatting. Might need to fix conflicts because https://github.com/coral-xyz/anchor/pull/3078 also added an entry to the "Breaking" section.

```suggestion

- client: Add `tokio` support to `RequestBuilder` with `async` feature ([#3057](https://github.com/coral-xyz/anchor/pull/3057])).
```